kdd-2024/ibis-tutorial-KDD2024.md

```diff
@@ -0,0 +1,286 @@
+
+- Ibis docs: https://ibis-project.org/
+- Ibis repository: https://github.com/ibis-project/ibis
+- Ibis blogs: https://ibis-project.org/posts
+- Ibis tutorial PyData NYC 2023: https://github.com/ibis-project/ibis-tutorial
+- The composable codex: https://voltrondata.com/codex
+
+
+# Strategies that you plan to employ to encourage audience participation and interactivity throughout the tutorial presentation
+
+Note: I adapted this based on my experience (and probably Gil's too) from Software
+Carpentry, we used some of these in the past. 
+
+To encourage audience participation and interactivity throughout the tutorial 
+presentation, we will employ several strategies following the general practices 
+use by [Software Carpentry](https://software-carpentry.org/), as well as other 
+practices that the instructors have found useful. These strategies include:
+
+- **Sticky Notes for Status Flags:** Each learner will be provided with two 
+sticky notes of different colors, such as red and green. Learners can use these 
+to discreetly indicate their status: green for completion or assistance needed, 
+and red for encountering problems. This method allows learners to continue 
+working while signaling for help, facilitating a smoother learning process for 
+everyone.
+
+- **Minute Cards for Feedback:** Before each break, learners will take a minute 
+to write one positive aspect on a green sticky note and one area for improvement 
+or question on a red sticky note. This anonymous feedback allows instructors to 
+adjust the pace and content of the tutorial based on learners' needs and interests.
+
+- **Pair Programming:** Pairing learners encourages collaborative problem-solving, 
+clarification of concepts, and discussion of shared interests. Seating arrangements 
+will facilitate pairing, with learners switching partners periodically to maximize
+interaction and prevent anyone from feeling isolated.
+
+- **Collaborative space to ask questions:** We will use the conference slack, or 
+provide a channel for the students to ask questions in a written format while the 
+tutorial is being performed. This lets us capture more questions, given that 
+sometimes not everyone feels comfortable asking it out loud.
+
+- **Peak Rule for Lesson Endings:** We will aim to end each lesson/section on a 
+high note, leaving learners with a positive impression of the session. By focusing
+on the most intense and final moments, we enhance the overall learning experience
+and leave participants feeling motivated and satisfied.
+
+- **One Up, One Down Feedback:** At the end of the day, learners will provide
+alternating positive and negative feedback points about the day's session. This 
+structured feedback encourages participants to express their honest opinions, 
+helping instructors to address concerns effectively.
+
+
+# A brief discussion of the potential societal impacts of your tutorial
+
```

Comment:
I took a crack at this, let me know if I over did it. Happy to cut some things over.